tools/Init2.ps1

```diff
@@ -0,0 +1,32 @@
+	##copy "E:\Git\PS\src\Package\Debug\ResourceManager\AzureResourceManager\AzureRM.Resources\Microsoft.Azure.Commands.ResourceManager.Cmdlets.*" "C:\Program Files\WindowsPowerShell\Modules\AzureRM.Resources\0.10.0\"
```

Comment:
@rnikhilkumar This looks like a security breach, as this is a credential exposed on a public repo, even though it's an internal credential.  Please remove, and you may need to report this to your security team